app/components/dashboard/diabetes/bs_over200_graph_component.html.erb

```diff
@@ -6,7 +6,7 @@
       <%= title.tooltip({ "Blood sugar 200-299 numerator" => t("bs_over_200_copy.bs_200_to_299.numerator"),
                           "Blood sugar ≥300 numerator" => t("bs_over_200_copy.bs_over_300.numerator"),
                           "Denominator" => t(denominator_copy, region_name: @region.name) }) %>
-      <%= title.ltfu_toggle(id: 'controlledGraphLtfuToggle', enabled: with_ltfu) %>
+      <%= title.ltfu_toggle(id: 'notControlledGraphLtfuToggle', enabled: with_ltfu) %>
```

Comment:
I've had to rename the diabetes toggles, when multiple things have the same ID only the first has the effect applied.

I cant see any other references to the ID in the code - hopefully this will not effect any of the data?

Reply:
This looks like the right thing to do. The ID should definitely be different. It doesn't look like this is referenced elsewhere. Paging @PriyangaPKini to check again, since I'm not that familiar with this part of the code.

[nit] The BP not controlled graph has the ID `uncontrolledGraphLtfuToggle`, perhaps this could have the same name for symmetry.